### ISSUE TITLE
[Proposal] Socio-Economic Vulnerability layers are not hazard-related

### ISSUE THREAD

**Issue by matamadio** [GFDRR/rdl-standard#238]
## What is the context or reason for the change?

There vulnerability component has several mandatory fields meant to describe a vulnerability function, e.g. the type of hazard and exp they apply to.
In the same component, we also aim to accomodate _se_category_ which is a different kind of object; not a function but a geospatial layer or just a table with location values. Some examples:

- Relative Wealth Index (spatial wealth distribution)
  <img src="https://gfdrr.github.io/CCDR-tools/_images/vln_rwi.jpg" width=50%>
- Vulnerable population (census attribues by adm unit)

Since these aren't model that connect hazard and exp values, the mandatory attributes of the schema can't be met:
![plantuml-cad5f6e1c2b28c2002f404123aeb540fc0b80a46 1](https://github.com/GFDRR/rdl-standard/assets/44863827/05f9531c-7757-49ad-a296-c826c993d4da)

## What is your proposed change?
I don't see many options. Either:

- put all fields as non-mandatory. That would not be optimal when proper vln functions are entered.
- separate the se_category from the rest of the vln schema alltogether

Suggestions are welcome!

**Comment by duncandewhurst** [comment]
From a non-expert point of view, it seems to me like that type of data better fits the description of `exposure`:

> Metadata that is specific to datasets that describe the situation of people, infrastructure, housing, production capacities and other tangible human assets located in hazard-prone areas.

Would it make sense to model those as exposure datasets?

**Comment by matamadio** [comment]
There's no right answer; indeed these are properties of exposure, but they are used to model its vulnerability (see [paper example](https://www.sciencedirect.com/science/article/pii/S2666592123000525https://www.sciencedirect.com/science/article/pii/S2666592123000525)).
It might be probably to fit in the exposure schema, in terms of data type and associated properties.

**Comment by duncandewhurst** [comment]
From today's check-in call, agreed that this requires further discussion and won't be done as part of 0.2.

**Comment by duncandewhurst** [comment]
From today's check-in call, agreed that this requires further discussion and won't be done as part of 0.2.

**Comment by stufraser1** [comment]
I agree it could fit into `exposure`, and #256, which creates `exposure_category` for social / economic indexes could address this, by allowing these index values as exposure category.

My suggestion: when data are clearly a social indicator like relative wealth index, it should be put into `exposure`.
When a vulnerability index combining multiple factors has been created to measure vulnerability specifically to one or more hazards, it can go into `vulnerability`.

**Comment by matamadio** [comment]
This issue is still relevant and needs to be discussed. There is no proper place for vulnerability indices atm and they don't seem to fit the exposure schema very well as they are applied in Vulnerability analysis, hence they should be part of the V schema.

**Comment by matamadio** [comment]
Solved by #395 #323
